_po/ja/how-to/laravel.po

```diff
@@ -479,9 +703,16 @@ msgid ""
 "\t</div>\n"
 "```"
 msgstr ""
+"```php\n"
+"<!-- Line 25 -->\n"
+"\t<div class=\"w-full sm:max-w-md mt-6 px-6 py-4 bg-white shadow-md overflow-hid"
+"den sm:rounded-lg\">\n"
+"\t\t{{ $slot }}\n"
+"\t</div>\n"
+"```"
 
 msgid "{% endraw %}"
-msgstr ""
+msgstr "{% endraw %}"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.